var/spack/repos/builtin/packages/geant4/package.py

```diff
@@ -107,7 +117,7 @@ def cmake_args(self):
 
         # Core options
         options = [
-            '-DGEANT4_BUILD_CXXSTD=c++{0}'.format(
+            '-DGEANT4_BUILD_CXXSTD={0}'.format(
```

Comment:
Is this a fix from what was there previously, or is it different for newer versions? If it's a fix, then this argument can be written as:
```python
self.define_from_variant('GEANT4_BUILD_CXXSTD', 'cxxstd')
```

Reply:
Yep, it's a fix - 10.7 onwards drop the "c++XY" form, but the "XY" form has always been allowed. Variant form added!